content/en/real_user_monitoring/rum_without_limits/_index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: RUM without Limits
+description: Keep only the RUM data you need while maintaining full visibility of performance metrics for your applications.
+private: true
+further_reading:
+  - link: '/real_user_monitoring/rum_without_limits/retention_filters'
+    tag: Documentation
+    text: Retain Data with Retention Filters
+  - link: '/real_user_monitoring/guide/retention_filter_best_practices/'
+    tag: Guide
+    text: Retention Filter Best Practices
+  - link: '/real_user_monitoring/rum_without_limits/metrics'
+    tag: Documentation
+    text: Analyze Performance with Metrics
+---
+
+{{< img src="real_user_monitoring/rum_without_limits/overview-test.png" alt="Estimated usage metrics details side panel" style="width:90%" >}}
+
+## Overview
+
+RUM without Limits provides you flexibility over your RUM sessions volumes by decoupling session data ingestion from indexing. This enables you to:
+
+- Dynamically set retention filters from the UI without upfront sampling decisions or code changes
+- Focus on high-impact sessions for improved visibility and troubleshooting of errors, crashes and high latencies
+- Retain sessions with errors or performance issues and discard less significant ones, such as ones with few user interactions
+
+Even if you retain only a fraction of your sessions, Datadog provides performance metrics for all ingested sessions. This ensures an accurate, long-term overview of application health and performance, even if only a fraction of session data is retained.
+
+This page identifies key components of RUM without Limits that can help you manage your RUM sessions volumes within your observability budget.
+
+## Setting up RUM without Limits
+
+### For new applications
+
+To get started with RUM without Limits for new applications, at the [instrumentation][1] step:
+
+1. Ensure the `sessionSampleRate` is set to 100%. This step is required.
+
+2. Choose a `sessionReplaySampleRate` that meets your observability needs.
+
+   **Note**: All replays are kept and billed.
+3. For applications with the [APM integration enabled][2], set the percentage of traces for which you want to make the correlation with APM traces with `traceSampling`.
+
+   <div class="alert alert-warning">Steps 3-4 can significantly impact APM traces ingestion.</div>
+
+4. Enable `traceContextInjection: sampled` to defer sampling decisions to backend tracers.
```

Comment:
@maxime-matheron I just realized that there aren't steps here to redeploy the application. Would the steps be the same as below for existing applications?